decidim-core/spec/lib/exporters/csv_spec.rb

```diff
@@ -37,5 +41,22 @@ def serialize
         expect(data[0]["serialized_name/ca"]).to eq("foocat")
       end
     end
+
+    describe "export sanitizer" do
```

Comment:
Awesome! 
Can you also add the specs to `decidim-core/spec/lib/exporters/excel_spec.rb` :pray: ?

Reply:
Hi @andreslucena 
do you mean to sanitize excel export + add new excel specs too ?

Reply:
I think the sanitization wouldn't be necessary as the Decidim::Exporters::Excel class already [inherits from CSV](https://github.com/decidim/decidim/blob/69e9271f23a6588df3972b38cd0121ae0b01e7b0/decidim-core/lib/decidim/exporters/excel.rb#L16), but as this attack could also be exploited with this exporter it'd be good to have it with a spec.

Reply:
Hi @andreslucena 
adding some specs to excel, I saw that even though Decidim::Exporters::Excel inherits from CSV, the export method is overwritten, so I added the sanitize check at this point as well.

Reply:
Great, I didn't catch that :clap: :clap: